src/lib/cli.ts

```diff
@@ -61,6 +69,22 @@ const setUpUserFeedback = (sonarInstance: Sonar, spinner: IORA) => {
     });
 };
 
+type Update = { // eslint-disable-line no-unused-vars
+    latest: string,
+    current: string,
+    type: string,
+    name: string
+};
+
+const notifyNewVersion = (update: Update) => {
+    const changelogUrl: string = `https://sonarwhal.com/about/changelog.html`;
+    // No indentation due to the use of `\` to avoid new line.
+    // https://stackoverflow.com/a/35428171
+    const message: string = `Update available ${chalk.dim(update.current)}${chalk.reset(' → ')}${chalk.green(update.latest)}\nRun ${chalk.cyan(`npm i -g ${update.name}`)} to update\nSee ${chalk.red(changelogUrl)} for update details`;
```

Comment:
@alrra How about using [detect-installed](https://github.com/charlike/detect-installed) to see if sonar was installed globally first. If yes, then add the `-g` flag in there?